app/User.php

```diff
@@ -12,4 +12,17 @@ class User extends Authenticatable
     protected $fillable = ['name', 'email', 'github_id', 'avatar', 'token'];
 
     protected $hidden = ['password', 'remember_token'];
+
+    public function getVerifyHash()
+    {
+        return md5($this->github_id . $this->token);
+    }
+
+    public function getUnsubscribeUrl()
+    {
+        return 'https://giscus.co/unsubscribe?' . http_build_query([
```

Comment:
can we build this with the `route()` helper so we're not hard coding either the domain or the path?

Reply:
I originally had the `route()` helper but removed it because the `url` in `app.php` wasn't set, and `route()` would either require a change there and/or in `.env`, which I didn't want to make you do as a prerequisite to merging the PR.